2023-08-26 17:12:11:wordhoard.utilities.request_html:INFO: --------------------------------------------------------------------------------
2023-08-26 17:12:11:wordhoard.utilities.request_html:INFO: The requested URL is protected by Cloudflare's DDoS mitigation service.
2023-08-26 17:12:11:wordhoard.utilities.request_html:INFO: Requested URL: https://www.collinsdictionary.com/dictionary/english-thesaurus/eat
2023-08-26 17:12:11:wordhoard.utilities.request_html:INFO: --------------------------------------------------------------------------------
2023-08-26 17:12:12:wordhoard.synonyms:ERROR: An AttributeError occurred in the following code segment:
2023-08-26 17:12:12:wordhoard.synonyms:ERROR:   File "C:\Users\ML\AppData\Local\Programs\Python\Python311\Lib\site-packages\wordhoard\synonyms.py", line 564, in _query_thesaurus_com
    for list_item in word_container.find('ul').find_all('li'):
                     ^^^^^^^^^^^^^^^^^^^

2023-08-26 17:12:13:wordhoard.synonyms:ERROR: A KeyError occurred in the following code segment:
2023-08-26 17:12:13:wordhoard.synonyms:ERROR:   File "C:\Users\ML\AppData\Local\Programs\Python\Python311\Lib\site-packages\wordhoard\synonyms.py", line 490, in _query_synonym_com
    self._update_cache(part_of_speech_category, synonyms_list)
  File "C:\Users\ML\AppData\Local\Programs\Python\Python311\Lib\site-packages\wordhoard\synonyms.py", line 135, in _update_cache
    caching.insert_word_cache_synonyms(self._word, pos_category, synonyms)
  File "C:\Users\ML\AppData\Local\Programs\Python\Python311\Lib\site-packages\wordhoard\utilities\caching.py", line 65, in insert_word_cache_synonyms
    temporary_dict_synonyms[word][pos_category] += deduplicated_values
    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^

2023-08-26 17:12:14:wordhoard.antonyms:INFO: Thesaurus.com had no antonym reference for the word eat
2023-08-26 17:13:56:wordhoard.utilities.request_html:INFO: --------------------------------------------------------------------------------
2023-08-26 17:13:56:wordhoard.utilities.request_html:INFO: The requested URL is protected by Cloudflare's DDoS mitigation service.
2023-08-26 17:13:56:wordhoard.utilities.request_html:INFO: Requested URL: https://www.collinsdictionary.com/dictionary/english-thesaurus/eat
2023-08-26 17:13:56:wordhoard.utilities.request_html:INFO: --------------------------------------------------------------------------------
2023-08-26 17:13:56:wordhoard.synonyms:ERROR: An AttributeError occurred in the following code segment:
2023-08-26 17:13:56:wordhoard.synonyms:ERROR:   File "C:\Users\ML\AppData\Local\Programs\Python\Python311\Lib\site-packages\wordhoard\synonyms.py", line 564, in _query_thesaurus_com
    for list_item in word_container.find('ul').find_all('li'):
                     ^^^^^^^^^^^^^^^^^^^

2023-08-26 17:13:58:wordhoard.synonyms:ERROR: A KeyError occurred in the following code segment:
2023-08-26 17:13:58:wordhoard.synonyms:ERROR:   File "C:\Users\ML\AppData\Local\Programs\Python\Python311\Lib\site-packages\wordhoard\synonyms.py", line 490, in _query_synonym_com
    self._update_cache(part_of_speech_category, synonyms_list)
  File "C:\Users\ML\AppData\Local\Programs\Python\Python311\Lib\site-packages\wordhoard\synonyms.py", line 135, in _update_cache
    caching.insert_word_cache_synonyms(self._word, pos_category, synonyms)
  File "C:\Users\ML\AppData\Local\Programs\Python\Python311\Lib\site-packages\wordhoard\utilities\caching.py", line 65, in insert_word_cache_synonyms
    temporary_dict_synonyms[word][pos_category] += deduplicated_values
    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^

2023-08-26 17:13:58:wordhoard.antonyms:INFO: Thesaurus.com had no antonym reference for the word eat
2023-08-26 17:15:07:wordhoard.utilities.request_html:INFO: --------------------------------------------------------------------------------
2023-08-26 17:15:07:wordhoard.utilities.request_html:INFO: The requested URL is protected by Cloudflare's DDoS mitigation service.
2023-08-26 17:15:07:wordhoard.utilities.request_html:INFO: Requested URL: https://www.collinsdictionary.com/dictionary/english-thesaurus/eat
2023-08-26 17:15:07:wordhoard.utilities.request_html:INFO: --------------------------------------------------------------------------------
2023-08-26 17:15:08:wordhoard.synonyms:ERROR: An AttributeError occurred in the following code segment:
2023-08-26 17:15:08:wordhoard.synonyms:ERROR:   File "C:\Users\ML\AppData\Local\Programs\Python\Python311\Lib\site-packages\wordhoard\synonyms.py", line 564, in _query_thesaurus_com
    for list_item in word_container.find('ul').find_all('li'):
                     ^^^^^^^^^^^^^^^^^^^

2023-08-26 17:15:09:wordhoard.synonyms:ERROR: A KeyError occurred in the following code segment:
2023-08-26 17:15:09:wordhoard.synonyms:ERROR:   File "C:\Users\ML\AppData\Local\Programs\Python\Python311\Lib\site-packages\wordhoard\synonyms.py", line 490, in _query_synonym_com
    self._update_cache(part_of_speech_category, synonyms_list)
  File "C:\Users\ML\AppData\Local\Programs\Python\Python311\Lib\site-packages\wordhoard\synonyms.py", line 135, in _update_cache
    caching.insert_word_cache_synonyms(self._word, pos_category, synonyms)
  File "C:\Users\ML\AppData\Local\Programs\Python\Python311\Lib\site-packages\wordhoard\utilities\caching.py", line 65, in insert_word_cache_synonyms
    temporary_dict_synonyms[word][pos_category] += deduplicated_values
    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^

2023-08-26 17:15:10:wordhoard.antonyms:INFO: Thesaurus.com had no antonym reference for the word eat
2023-08-26 17:16:42:wordhoard.utilities.request_html:INFO: --------------------------------------------------------------------------------
2023-08-26 17:16:42:wordhoard.utilities.request_html:INFO: The requested URL is protected by Cloudflare's DDoS mitigation service.
2023-08-26 17:16:42:wordhoard.utilities.request_html:INFO: Requested URL: https://www.collinsdictionary.com/dictionary/english-thesaurus/eat
2023-08-26 17:16:42:wordhoard.utilities.request_html:INFO: --------------------------------------------------------------------------------
2023-08-26 17:16:43:wordhoard.synonyms:ERROR: An AttributeError occurred in the following code segment:
2023-08-26 17:16:43:wordhoard.synonyms:ERROR:   File "C:\Users\ML\AppData\Local\Programs\Python\Python311\Lib\site-packages\wordhoard\synonyms.py", line 564, in _query_thesaurus_com
    for list_item in word_container.find('ul').find_all('li'):
                     ^^^^^^^^^^^^^^^^^^^

2023-08-26 17:16:45:wordhoard.synonyms:ERROR: A KeyError occurred in the following code segment:
2023-08-26 17:16:45:wordhoard.synonyms:ERROR:   File "C:\Users\ML\AppData\Local\Programs\Python\Python311\Lib\site-packages\wordhoard\synonyms.py", line 490, in _query_synonym_com
    self._update_cache(part_of_speech_category, synonyms_list)
  File "C:\Users\ML\AppData\Local\Programs\Python\Python311\Lib\site-packages\wordhoard\synonyms.py", line 135, in _update_cache
    caching.insert_word_cache_synonyms(self._word, pos_category, synonyms)
  File "C:\Users\ML\AppData\Local\Programs\Python\Python311\Lib\site-packages\wordhoard\utilities\caching.py", line 65, in insert_word_cache_synonyms
    temporary_dict_synonyms[word][pos_category] += deduplicated_values
    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^

2023-08-26 17:16:45:wordhoard.antonyms:INFO: Thesaurus.com had no antonym reference for the word eat
2023-08-26 17:19:21:wordhoard.utilities.request_html:INFO: --------------------------------------------------------------------------------
2023-08-26 17:19:21:wordhoard.utilities.request_html:INFO: The requested URL is protected by Cloudflare's DDoS mitigation service.
2023-08-26 17:19:21:wordhoard.utilities.request_html:INFO: Requested URL: https://www.collinsdictionary.com/dictionary/english-thesaurus/home
2023-08-26 17:19:21:wordhoard.utilities.request_html:INFO: --------------------------------------------------------------------------------
2023-08-26 17:19:22:wordhoard.synonyms:ERROR: An AttributeError occurred in the following code segment:
2023-08-26 17:19:22:wordhoard.synonyms:ERROR:   File "C:\Users\ML\AppData\Local\Programs\Python\Python311\Lib\site-packages\wordhoard\synonyms.py", line 564, in _query_thesaurus_com
    for list_item in word_container.find('ul').find_all('li'):
                     ^^^^^^^^^^^^^^^^^^^

2023-08-26 17:19:24:wordhoard.antonyms:INFO: Thesaurus.com had no antonym reference for the word home
2023-08-26 17:20:05:wordhoard.utilities.request_html:INFO: --------------------------------------------------------------------------------
2023-08-26 17:20:05:wordhoard.utilities.request_html:INFO: The requested URL is protected by Cloudflare's DDoS mitigation service.
2023-08-26 17:20:05:wordhoard.utilities.request_html:INFO: Requested URL: https://www.collinsdictionary.com/dictionary/english-thesaurus/home
2023-08-26 17:20:05:wordhoard.utilities.request_html:INFO: --------------------------------------------------------------------------------
2023-08-26 17:20:05:wordhoard.synonyms:ERROR: An AttributeError occurred in the following code segment:
2023-08-26 17:20:05:wordhoard.synonyms:ERROR:   File "C:\Users\ML\AppData\Local\Programs\Python\Python311\Lib\site-packages\wordhoard\synonyms.py", line 564, in _query_thesaurus_com
    for list_item in word_container.find('ul').find_all('li'):
                     ^^^^^^^^^^^^^^^^^^^

2023-08-26 17:20:07:wordhoard.antonyms:INFO: Thesaurus.com had no antonym reference for the word home
2023-08-26 17:20:31:wordhoard.utilities.request_html:INFO: --------------------------------------------------------------------------------
2023-08-26 17:20:31:wordhoard.utilities.request_html:INFO: The requested URL is protected by Cloudflare's DDoS mitigation service.
2023-08-26 17:20:31:wordhoard.utilities.request_html:INFO: Requested URL: https://www.collinsdictionary.com/dictionary/english-thesaurus/home
2023-08-26 17:20:31:wordhoard.utilities.request_html:INFO: --------------------------------------------------------------------------------
2023-08-26 17:20:32:wordhoard.synonyms:ERROR: An AttributeError occurred in the following code segment:
2023-08-26 17:20:32:wordhoard.synonyms:ERROR:   File "C:\Users\ML\AppData\Local\Programs\Python\Python311\Lib\site-packages\wordhoard\synonyms.py", line 564, in _query_thesaurus_com
    for list_item in word_container.find('ul').find_all('li'):
                     ^^^^^^^^^^^^^^^^^^^

2023-08-26 17:20:34:wordhoard.antonyms:INFO: Thesaurus.com had no antonym reference for the word home
2023-08-26 17:21:03:wordhoard.utilities.request_html:INFO: --------------------------------------------------------------------------------
2023-08-26 17:21:03:wordhoard.utilities.request_html:INFO: The requested URL is protected by Cloudflare's DDoS mitigation service.
2023-08-26 17:21:03:wordhoard.utilities.request_html:INFO: Requested URL: https://www.collinsdictionary.com/dictionary/english-thesaurus/book
2023-08-26 17:21:03:wordhoard.utilities.request_html:INFO: --------------------------------------------------------------------------------
2023-08-26 17:21:04:wordhoard.synonyms:ERROR: An AttributeError occurred in the following code segment:
2023-08-26 17:21:04:wordhoard.synonyms:ERROR:   File "C:\Users\ML\AppData\Local\Programs\Python\Python311\Lib\site-packages\wordhoard\synonyms.py", line 564, in _query_thesaurus_com
    for list_item in word_container.find('ul').find_all('li'):
                     ^^^^^^^^^^^^^^^^^^^

2023-08-26 17:21:06:wordhoard.antonyms:INFO: Thesaurus.com had no antonym reference for the word book
2023-08-26 17:23:50:wordhoard.utilities.request_html:INFO: --------------------------------------------------------------------------------
2023-08-26 17:23:50:wordhoard.utilities.request_html:INFO: The requested URL is protected by Cloudflare's DDoS mitigation service.
2023-08-26 17:23:50:wordhoard.utilities.request_html:INFO: Requested URL: https://www.collinsdictionary.com/dictionary/english-thesaurus/eat
2023-08-26 17:23:50:wordhoard.utilities.request_html:INFO: --------------------------------------------------------------------------------
2023-08-26 17:23:50:wordhoard.synonyms:ERROR: An AttributeError occurred in the following code segment:
2023-08-26 17:23:50:wordhoard.synonyms:ERROR:   File "C:\Users\ML\AppData\Local\Programs\Python\Python311\Lib\site-packages\wordhoard\synonyms.py", line 564, in _query_thesaurus_com
    for list_item in word_container.find('ul').find_all('li'):
                     ^^^^^^^^^^^^^^^^^^^

2023-08-26 17:23:50:wordhoard.synonyms:ERROR: A KeyError occurred in the following code segment:
2023-08-26 17:23:50:wordhoard.synonyms:ERROR:   File "C:\Users\ML\AppData\Local\Programs\Python\Python311\Lib\site-packages\wordhoard\synonyms.py", line 490, in _query_synonym_com
    self._update_cache(part_of_speech_category, synonyms_list)
  File "C:\Users\ML\AppData\Local\Programs\Python\Python311\Lib\site-packages\wordhoard\synonyms.py", line 135, in _update_cache
    caching.insert_word_cache_synonyms(self._word, pos_category, synonyms)
  File "C:\Users\ML\AppData\Local\Programs\Python\Python311\Lib\site-packages\wordhoard\utilities\caching.py", line 65, in insert_word_cache_synonyms
    temporary_dict_synonyms[word][pos_category] += deduplicated_values
    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^

2023-08-26 17:23:51:wordhoard.antonyms:INFO: Thesaurus.com had no antonym reference for the word eat
